open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.POW

```diff
@@ -546,34 +764,77 @@ PRINT TITLE="BANNO.NEWSUBCREATE.V1.POW"
        NEWLINE
        PRINT "   "+Q+"canOpenSubShare"+Q+": true,"
        NEWLINE
+
+[* RESULTS START
+*]
        PRINT "  "+Q+"results"+Q+": {"
        NEWLINE
-
+[* if new names are not allowed or if they are allowed but the max new names
+** are nset to 0 or and there are no eligible copy names then set new names
```

Comment:
I think you mean to say "are set to 0" instead of "nset" in the comment "but the max new names
** are nset to 0"